docs/tutorials/getting_started.md

```diff
@@ -61,110 +65,197 @@ directory:
+router                             active       1  sdcore-router-k8s             1.3/edge            33  10.152.183.49    no
+self-signed-certificates           active       1  self-signed-certificates      1.3/edge            33  10.152.183.153   no
+smf                                active       1  sdcore-smf-k8s                1.3/edge            37  10.152.183.147   no
+traefik                   2.10.4   active       1  traefik-k8s                   latest/stable      148  10.0.0.3         no
+udm                                active       1  sdcore-udm-k8s                1.3/edge            35  10.152.183.168   no
+udr                                active       1  sdcore-udr-k8s                1.3/edge            31  10.152.183.96    no
+upf                                active       1  sdcore-upf-k8s                1.3/edge            64  10.152.183.126   no
+webui                              active       1  sdcore-webui-k8s              1.3/edge            23  10.152.183.128   no
 
 Unit                         Workload  Agent  Address      Ports  Message
 amf/0*                       active    idle   10.1.182.23
 ausf/0*                      active    idle   10.1.182.18
 gnbsim/0*                    active    idle   10.1.182.50
-grafana-agent-k8s/0*         blocked   idle   10.1.182.51         logging-consumer: off, grafana-cloud-config: off
-mongodb-k8s/0*               active    idle   10.1.182.35         Primary
+grafana-agent/0*             blocked   idle   10.1.182.51         logging-consumer: off, grafana-cloud-config: off
+mongodb/0*                   active    idle   10.1.182.35         Primary
 nms/0*                       active    idle   10.1.182.2
 nrf/0*                       active    idle   10.1.182.53
 nssf/0*                      active    idle   10.1.182.48
 pcf/0*                       active    idle   10.1.182.46
 router/0*                    active    idle   10.1.182.57
 self-signed-certificates/0*  active    idle   10.1.182.56
 smf/0*                       active    idle   10.1.182.27
-traefik-k8s/0*               active    idle   10.1.182.40
+traefik/0*                   active    idle   10.1.182.40
 udm/0*                       active    idle   10.1.182.52
 udr/0*                       active    idle   10.1.182.39
 upf/0*                       active    idle   10.1.182.60
 webui/0*                     active    idle   10.1.182.33
 ```
 
-## 4. Deploy the 5G simulator
-
-Deploy the `sdcore-gnbsim-k8s` operator
-
-```console
-juju deploy sdcore-gnbsim-k8s gnbsim --trust --channel=beta
-```
-
-Integrate it to the AMF and the NMS:
-
-```console
-juju integrate gnbsim:fiveg-n2 amf:fiveg-n2
-juju integrate gnbsim:fiveg_gnb_identity nms:fiveg_gnb_identity
-```
-
-## 5. Configure the ingress
+## 6. Configure the ingress
 
 Configure Traefik to use an external hostname:
 
 ```console
-juju config traefik-k8s external_hostname=10.0.0.3.nip.io
+juju config traefik external_hostname=10.0.0.3.nip.io
```

Comment:
We seem to have a race condition problem here.  In my case, traefik took .4, not .3:
```
sdcore                          amf-external                         LoadBalancer   10.152.183.68    10.0.11.2     38412:32503/SCTP              35m
sdcore                          upf-external                         LoadBalancer   10.152.183.34    10.0.11.3     8805:31595/UDP                33m
sdcore                          traefik                              LoadBalancer   10.152.183.70    10.0.11.4     80:32753/TCP,443:31561/TCP    36m
```

Note that I used 10.0.11.2-10.0.11.4 as the range due to 10.0.0.0 being my home subnet